src/index.ts

```diff
@@ -124,20 +135,24 @@ export async function generate(
     allowUnusedLabels: true,
   };
 
-  const { libs, templateNames, pkg } = mergeWith(
+  const { libs, templateNames, pkg, partials } = mergeWith(
     {},
     ...Object.entries(frameworks)
       .filter(([_, framework]) => framework !== undefined)
       .map(([kind, framework]) => getGenerator(runtime, kind, framework)),
     (a, b) => isArray(a) ? a.concat(b) : undefined
   );
   const libContents = await Promise.all(libs.map((n) => readFile(path.join(libPath, n), 'utf-8')));
+  const partialContents = zipObject(
```

Comment:
I think it would be somewhat simpler to write using readFileSync...

Reply:
But not that much simpler, I'm leaving it.
It's not critical either way.